Makefile.generate_proto

```diff
@@ -0,0 +1 @@
+build/Makefile.generate_proto
```

Comment:
Pls add one Readme file with all the support provided by this make file

Reply:
The issue is tracked in #54